homeassistant/components/here_travel_time/config_flow.py

```diff
@@ -249,39 +255,47 @@ async def async_step_import(
         )
 
     def _transform_import_input(
-        self, user_input
+        self, import_input
```

Comment:
HA style wants typing to be all or nothing so want to add a type for this argument.

```suggestion
        self, import_input: dict[str, Any]
```